src/vs/workbench/contrib/welcomeGettingStarted/browser/media/gettingStarted.css

```diff
@@ -615,6 +615,8 @@
 
 .monaco-workbench .part.editor>.content .gettingStartedContainer .index-list.getting-started .button-link {
 	margin: 0;
+	outline: none;
+	user-select: none;
```

Comment:
AFAIK we allow users to select links elsewhere

<img width="334" alt="CleanShot 2023-04-28 at 12 51 35@2x" src="https://user-images.githubusercontent.com/25163139/235240256-db3ea680-d337-4cbf-ae4a-b4522065de02.png">

Reply:
Can you please give the steps to reproduce this?

Reply:
Go to the search view without a folder open

Reply:
> # - - [ ] @~~_[]()****_~~

Reply:
0e24a32 applies the same behavior to the "Open Folder" text button.

Reply:
My point was that we should not be making this change given that it would be the one link button that has special cases. I'm ok with the change to align the link overall, but I'd prefer not to change the link selection behavior.

Reply:
I will revert those changes for now then.

The link button behavior is somewhat inconsistent. In this case, text selection is prevented but the outline is not:

![PixelSnap 2023-04-28 at 22 17 13@2x](https://user-images.githubusercontent.com/2276355/235245646-c5cd2da2-b1b1-4af9-94a9-5815039fd513.png)

Reply:
Hi @daviddossett, only the link alignment is now part of this pull request. How to proceed?